src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/constants/style.scss";
+
+$icon-text--gap: 20px;
+$columns-list--padding-left: $icon-text--gap + 50px;
+$list-style-type: "– ";
+
+.export-board__hint-hidden-columns {
+  margin-top: $margin--large;
+  padding: 0 $padding--large 0 $padding--large;
+
+  font-size: $text-size--medium;
+}
+
+.hint-hidden-columns__info-container {
+  display: flex;
+  align-items: center;
+}
+
+.hint-hidden-columns__info-icon {
+  width: $icon--huge;
+  height: $icon--huge;
+
+  margin-right: $icon-text--gap;
+
+  g g,
+  line {
+    stroke: $color-warning-red;
+  }
+}
```

Comment:
The icon itself has predefined strokes, if you remove them, you can apply the stroke to the icon class instead of specifically applying it to the line itself.

<img width="674" alt="image" src="https://user-images.githubusercontent.com/44020029/189611126-2bc2740b-cde0-43fc-b2f9-66054bf3692a.png">

Reply:
oh lol of course, why didn't I think of this 🤦 😂